staging/publishing/rules.yaml

```diff
@@ -216,8 +216,6 @@ rules:
       branch: master
     - repository: code-generator
       branch: master
-    - repository: component-base
-      branch: master
```

Comment:
I think it was ok to have extra deps listed in rules with godeps, but with go modules it leads to an awkward go.mod file: https://github.com/kubernetes/sample-controller/blob/d4d703847b23378cf5779831533cd96175d95632/go.mod#L28

Reply:
Is this check by the verify script now?

Reply:
Yes, the "extra deps" check.